### ISSUE TITLE
[BUG] when Linux has no batteries, EWW_BATTERY is not valid JSON

### ISSUE THREAD

**Issue by jokeyrhyme** [elkowar/eww#373]
### Checklist before submitting an issue

- [X] I have searched through the existing [closed and open issues](https://github.com/elkowar/eww/issues?q=is%3Aissue) for eww and made sure this is not a duplicate
- [X] I have specifically verified that this bug is not a common [user error](https://github.com/elkowar/eww/issues?q=is%3Aissue+label%3Ano-actual-bug+is%3Aclosed)
- [X] I am providing as much relevant information as I am able to in this bug report (Minimal config to reproduce the issue for example, if applicable)

### Description of the bug

I use the same eww.yuck on both a gaming PC (no batteries), and a laptop: https://gitlab.com/jokeyrhyme/dotfiles/-/blob/main/config/eww/eww.yuck

I've noticed that on my PC, with no batteries, the EWW_BATTERY value is not valid JSON, which makes working with it tricky

### Reproducing the issue

```
$ eww state
EWW_BATTERY: , "total_avg": NaN}
EWW_RAM: {"total_mem": 33588690, "free_mem": 19586744, "total_swap": 240031621, "free_swap": 240031621, "available_mem": 27843256, "used_mem": 5745432, "used_mem_perc": 17.105259}
```

You can see here that EWW_RAM is valid JSON, but EWW_BATTERY is truncated, and I'm also not sure if `NaN` is a valid JSON number

### Expected behaviour

I'm not quite sure what to expect, I suppose this is a question of design

- should EWW_BATTERY be completely absent when there is no battery ?
- should it be an empty string ?
- should it be JSON like `{ "total_avg": null }` ?

### Additional context

I believe this line might need some adjustment: https://github.com/elkowar/eww/blob/master/crates/eww/src/config/system_stats.rs#L153

I might have an attempt at this myself if I get some free time

**Comment by legendofmiracles** [comment]
Can you upload the output of `tree /sys/class/power_supply` somewhere? We could check if the directory is empty. IMHO it should just completely be `null` when there is no battery, there isn't much use to having the json values in the first place then.

**Comment by jokeyrhyme** [comment]
```
$ ls /sys/class/power_supply
[no output at all]
```

That directory doesn't even exist for me when there's no battery

For the record, that's with this kernel:

```
$ uname -a
Linux hellmouth 5.15.10-arch1-1 #1 SMP PREEMPT Fri, 17 Dec 2021 11:17:37 +0000 x86_64 GNU/Linux
```

**Comment by andornaut** [comment]
> [...] it should just completely be `null` then, when there is no battery, there isn't much use to having the json values in the first place.

I implemented this suggestion in PR #425.
